projects/storefrontapp-e2e-cypress/cypress/support/select-user-menu-option.commands.ts

```diff
@@ -31,7 +31,8 @@ Cypress.Commands.add(
     }
 
     cy.get(
-      'cx-login > cx-page-slot > cx-navigation > cx-navigation-ui > nav > ul > li > div > ul'
+      'cx-login > cx-page-slot > cx-navigation > cx-navigation-ui > nav > ul > li > div > ul',
+      { timeout: Cypress.config('defaultCommandTimeout') + 20000 }
```

Comment:
I don't think we need this anymore as I think it was complaining before due to the fact we had that Login error for unauthorized users, which was fixed [here](https://github.com/SAP/spartacus/commit/c1a04a3d2286b031f72aac6931ee71bfb51d0149)

Reply:
Moreover, the default timeout for the CI e2e tests is 30s, which is already long enough to wait for a response to at least come back from the backend